src/main/java/com/drinchev/projectlabel/resources/ui/ProjectLabelStatusBarWidget.java

```diff
@@ -34,6 +35,7 @@ public class ProjectLabelStatusBarWidget extends JButton implements CustomStatus
 
     private static final int HORIZONTAL_PADDING = 18;
     private static final int VERTICAL_PADDING = 2;
+    private static final int VERTICAL_MARGIN = Registry.is("ide.experimental.ui") ? 5 : 3;
```

Comment:
Suggestion: Maybe we should create a `isNewUI()` method in one of the util classes (implementation based on the `getStatusBarItemFont()` approach?) and use that wherever we need to switch some styling based on that fact?

Reply:
Good idea ;)